photon-client/src/types/PhotonTrackingTypes.ts

```diff
@@ -1,3 +1,26 @@
+export interface Quaternion {
+  X: number;
+  Y: number;
+  Z: number;
+  W: number;
+}
+
+export interface Translation3d {
+  x: number;
+  y: number;
+  z: number;
+}
+
+export interface Rotation3d {
+  quaternion: Quaternion;
+}
+
+export interface Pose3d {
+  translation: Translation3d;
+  rotation: Rotation3d;
+}
+
+// TODO update backend to serialize this using correct layout
```

Comment:
Should this be done now as well or a separate PR?

Reply:
I need Pose3d for optimisedCameraToObject in CameraCalibrationResult? But that change can live in another PR